tests/framework_tests/test_analytics.py

```diff
@@ -66,108 +69,102 @@ def setUp(self):
         }
 
     def test_update_counters_file(self):
-
-        @analytics.update_counters('download:{target_id}:{fid}', db=self.db)
+        @analytics.update_counters('download:{target_id}:{fid}', db=None)
         def download_file_(**kwargs):
             return kwargs.get('node') or kwargs.get('project')
 
-        count = analytics.get_basic_counters('download:{0}:{1}'.format(self.node, self.fid), db=self.db)
+        count = analytics.get_basic_counters('download:{0}:{1}'.format(self.node._id, self.fid), db=None)
         assert_equal(count, (None, None))
 
         download_file_(node=self.node, fid=self.fid)
 
-        count = analytics.get_basic_counters('download:{0}:{1}'.format(self.node, self.fid), db=self.db)
+        count = analytics.get_basic_counters('download:{0}:{1}'.format(self.node._id, self.fid), db=None)
         assert_equal(count, (1, 1))
 
-        page = 'download:{0}:{1}'.format(self.node, self.fid)
+        page = 'download:{0}:{1}'.format(self.node._id, self.fid)
 
         session.data['visited'].append(page)
         download_file_(node=self.node, fid=self.fid)
 
-        count = analytics.get_basic_counters('download:{0}:{1}'.format(self.node, self.fid), db=self.db)
+        count = analytics.get_basic_counters('download:{0}:{1}'.format(self.node._id, self.fid), db=None)
```

Comment:
Yeah, modm's defaults `__str__` returns the string _id, so this change is appropriate. 👍